app/Http/Controllers/KebabPlaceController.php

```diff
@@ -73,10 +85,8 @@ public function show(Request $request, ?User $user): JsonResponse
         $kebabPlace->comments->each(function ($comment): void {
             $comment->is_owner = $comment->user_id === auth()->id();
         });
-        $ids = explode(", ", $kebabPlace->fillings);
-        $kebabPlace->fillings = Filling::whereIn("id", $ids)->get();
-        $ids = explode(", ", $kebabPlace->sauces);
-        $kebabPlace->sauces = Sauce::whereIn("id", $ids)->get();
+        $kebabPlace->fillings = Filling::whereIn("id", $kebabPlace->fillings)->get();
```

Comment:
```suggestion
        $kebabPlace->fillings = Filling::query()->whereIn("id", $kebabPlace->fillings)->get();
```

It's a good practise to use query(), linter formats code nicely

Reply:
I see that you added query() in some places, and in some you don't